pkg/game/board_test.go

```diff
@@ -40,7 +40,9 @@ func TestBoardGetTilePlacementsForReturnsEmptySliceWhenCityCannotBePlaced(t *tes
 					Tile: tiletemplates.SingleCityEdgeNoRoads().Rotate(2),
 					Pos:  elements.NewPosition(0, 1),
 				},
-				Meeple: elements.MeeplePlacement{Side: side.None},
+				Meeple: elements.MeeplePlacement{
+					Feature: feature.Feature{FeatureType: feature.None},
+				},
```

Comment:
Random idea: would it be possible to define some constant like "elements.NoMeeple"? This is a very verbose way to say a meeple isn't there.